### DISCUSSION TITLE
set buy order on steam market

### DISCUSSION THREAD

**Discussion by evgewakan** [chr233/ASFEnhance#88] [功能建议 / Ideas]
Sometimes there is a need to transfer the balance from one account to another, there are several options for this, but in my opinion the safest is placing orders and buying items with further transfer to the desired account.

I propose to add placing orders for the purchase of items on the steam market to the functionality.

Example:
BotA SetBuyOrder appid/name_items count price

**Comment by woctezuma** [comment]
If you want to transfer money via the Steam Market, wouldn't it be safer and simpler to use **a single item** which is not a commodity?
For instance, [Team Fortress 2 > Health and Hell Knife (Field-Tested)](https://steamcommunity.com/market/listings/440/Health%20and%20Hell%20Knife%20(Field-Tested)).
